test/e2e/run_e2e.sh

```diff
@@ -231,6 +241,37 @@ if [ "$stackset_e2e" = true ]; then
     E2E_NAMESPACE="${namespace}" ./stackset-e2e -test.parallel 20
 fi
 
+if [ "$loadtest_e2e" = true ]; then
+  >&2 echo "collect loadtest e2e data"
+  prometheus=$(kubectl -n loadtest-e2e get ing prometheus -o json | jq -r '.spec.rules[0].host')
+
+  >&2 echo "target prometheus: ${prometheus}"
+
+  # get data for the last 30m
+  curl -s -H"Accept: application/json" \
```

Comment:
curl supports `--data-urlencode "step=60"` that works together with `--get`, I think it could make the command cleaner

Reply:
no it would send the query in the body and IMO would not be cleaner this way, because you need extra handling for the query anyways, because the PromQL query consists of characters you need to extra handle anyways.

Reply:
With `--get`, `--data-urlencode` will do proper urlencoding of parameters into the url, not the body:
```sh
curl -v --get \
  --data-urlencode 'query=sum by(code) (rate(skipper_serve_host_count{application="e2e-vegeta"}[1m]))' \
  --data-urlencode "start=$(( $(date +%s) - (30*60) ))" \
  --data-urlencode "end=$(( $(date +%s) ))" \
  --data-urlencode "step=60" \
  "https://${prometheus}/api/v1/query_range"
```

Reply:
Interesting I tested `-X GET` and that is using the body, but `--get` is putting it into the query.

Reply:
fixed